code-challenges.js

```diff
@@ -7,47 +7,152 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in any two strings and returns the string with more characters. Use the two sets of test variables provided.
 
 // Pseudo code:
-
+//create a function that takes in two strings as arguments
+//conditional to check which is longer string
+//if string1 is longer return it else return the other string
+//
 // Set one:
 const fruit1 = "apple"
 const fruit2 = "banana"
 
 // Set two:
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
-
-
-// --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+//first method
+const longestVar = (str1,str2) => {
+    if(str1.length > str2.length){
+    return str1;
+} else {
+    return str2;
+}
+}
+console.log(longestVar(fruit1, fruit2))
+console.log(longestVar(fruit3,fruit4))
+// --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point.
+// Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: 
+//"42 is below boiling point",
+ //"350 is above boiling point", 
+ //"212 is at boiling point"
 
 // Pseudo code:
-
+//create a function that takes in a given number named it boilingPoint
+// if statement to check for anything 42 and below, if true return ...
+//else if the temp = 212 say its at boiling
+//else if temp is above 350 return is above boiling point
+// stretch can I do this with another shorter one line method? Would it be inerpolation?
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilngPoint = (num) => {
+    if (num < temp1 ){
+        return "temp is below boiling point"; 
+    }else if
+        (num = temp3){
+            return "is at boiling point";
+        } else if (num > temp2){ 
+            return "is above boiling point";
+        }
+    };
+
```

Comment:
This approach works with the given test variable. Recommend you refactor using 212 as the reference point. Remember that `=` is an assignment operator. `===` or `==` are the equality operators.